torq.sh

```diff
@@ -223,6 +221,18 @@ usage() {
   exit 1
  }
 
+if [[ $(printenv | grep SETENV) ]]; then
+  SETENV=$(printenv | grep SETENV | sed -e 's#.*=\(\)#\1#')                                         # set predefined environment
```

Comment:
why is this needed? If it's already set, it's all good. I think this should be a case of setting it if it's not set, otherwise leaving it e.g.

```bash
jmcmurray@homer ~ $ echo $SETENV  # not set

jmcmurray@homer ~ $ if [[ -z "$SETENV" ]]; then SETENV="test"; fi
jmcmurray@homer ~ $ echo $SETENV
test
jmcmurray@homer ~ $ SETENV=example #pre-set value
jmcmurray@homer ~ $ if [[ -z "$SETENV" ]]; then SETENV="test"; fi
jmcmurray@homer ~ $ echo $SETENV
example
```

Alternatively you may be able to use a default value expansion, e.g.

```bash
jmcmurray@homer ~ $ echo $SETENV
example
jmcmurray@homer ~ $ echo ${SETENV:-test}
example
jmcmurray@homer ~ $ unset SETENV
jmcmurray@homer ~ $ echo ${SETENV:-test}
test
```